lib/miq_automation_engine/engine/miq_ae_service_model_base.rb

```diff
@@ -29,7 +31,7 @@ def self.respond_to?(m, *args)
 
     # Expose the ActiveRecord find, all, count, and first
     def self.class_method_exposed?(m)
-      m.to_s.starts_with?('find_') || [:where, :find, :all, :count, :first].include?(m)
+      m.to_s.starts_with?('find_') || [:where, :find].include?(m)
```

Comment:
@kbrock 
This was a temporary solution till we get to REST. We still have customers using the find_by... so we can't remove it yet.